htdocs/landing/privacy.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <title>GOCDB Privacy Notice</title>
+  <link rel="stylesheet" type="text/css" href="home.css" />
+  <link rel="SHORTCUT ICON" href="Logo-1.4-FavIcon-32x32.regional.ico" />
+</head>
+
+<div class="wide_page_container" >
+  <div class="centre_box">
+    <div class="centrePageContainer">
+      <div class="policyText">
+        <!--Start of policy text output by PANDOC.-->
+        <h1 id="grid-operations-centre-database-privacy-notice">Grid Operations Centre Database Privacy Notice</h1>
+        <p>Grid Operations Centre Database (GOCDB) is a central registry to record information about the topology of the EGI e-Infrastructure. This includes information describing operations and resource centres, the type, network connection endpoint and downtimes of services, and the roles and contact information of users.</p>
+        <p>The Science and Technology Facilities Council (STFC) provides the GOCDB service on behalf of EGI.eu. This Privacy Notice describes what personal data is collected from you when you use GOCDB, why it is collected and how this data is used.</p>
+        <p>STFC is part of UK Research and Innovation (UKRI - <a href="https://stfc.ukri.org/">stfc.ukri.org</a>). UKRI and STFC comply with the requirements of the European General Data Protection Regulation with regard to the collection, storage, processing and disclosure of personal information and are committed to upholding the GDPR's core data protection principles. A full notice of UKRI's position with regards to Freedom of Information and the GDPR (EU) (2016/679) can be found at: <a href="http://www.ukri.org/freedom-of-information">www.ukri.org/freedom-of-information</a></p>
```

Comment:
The `(UKRI - stfc.ukri.org)` bit is confusing as you have elements with different purposes within the same brackets (abbreviation and hyperlink). The second mention of UKRI should use the full name and the abbreviation be included with that one.

Reply:
Agreed, it's messy. Follow UKRI branding guidelines more closely and split to two sentences. "STFC is part of UK Research and Innovation (UKRI). For more information visit stfc.ukri.org. UKRI and STFC comply .... "

Reply:
That works.